CMakeLists.txt

```diff
@@ -578,6 +578,10 @@ set(GUI_SOURCES
   src/scadlexer.cpp
   src/scintillaeditor.cpp
   src/settings.cc
+  src/Preferences.cc
+  src/initConfig.cc
+  src/ShortcutConfigurator.cc
```

Comment:
`ShortCutConfigurator.cc` needed to be added, but the other additions here are duplicated in this list.

By the way, the file list is now sorted alphabetically, where filenames starting with lowercase characters come before uppercase ones (similar to how github displays files in the code tree view, although I guess they do uppercase before lowercase).  Hope that makes sense.

Reply:
Ok, I have fixed that :)